test/fixtures/base/test-img/img.twig

```diff
@@ -0,0 +1,3 @@
+<!-- PATTERN START - /pattern-library/patterns/base/img -->
```

Comment:
This file is part of the files associated with the tests. This specific one is here to test that twig files are including other files. There isn't currently a test written to check image markup specifically.